docusaurus/docs/themes/transportation/shape-connectivity.mdx

```diff
@@ -0,0 +1,430 @@
+segment and the last coordinate is the "end". A segment is "oriented"
+away from the start and toward the end. The examples below show two
+segment geometries with identical coordinates, oriented in opposite
+directions.
+
+<div style={{ width: "49.5%", float: "left" }}>
+
+<figure>
+
+```yaml
+type: LineString
+coordinates:
+  - [1, 0]   # Start
+  - [0, 0]
+  - [-1, 0]  # End
+```
+
+<div style={{textAlign: 'center'}}>
+
+![West-oriented segment geometry](/img/transportation/oriented-west.svg)
+
+</div>
+
+<figcaption>
+
+*This segment geometry is oriented due west.*
+
+</figcaption>
+</figure>
+
+</div>
+
+<div style={{ width: "49.5%", float: "right" }}>
+
+<figure>
+
+```yaml
+type: LineString
+coordinates:
+  - [-1, 0]  # Start
+  - [0, 0]
+  - [1, 0]   # End
+```
+
+<div style={{textAlign: 'center'}}>
+
+![East-oriented segment geometry](/img/transportation/oriented-east.svg)
+
+</div>
+
+<figcaption>
+
+*This segment geometry is oriented due east.*
+
+</figcaption>
+</figure>
+
+</div>
+
+<div style={{ clear: "both" }}/>
+
+
+### Heading
+
+Travel along a segment's geometry can follow one of two possible
+*headings*: `forward` or `backward`. The `forward` heading proceeds
+toward the end of the segment; while the `backward` heading proceeds
+back toward the start of the segment.
+
+<div style={{textAlign: 'center'}}>
+
+<figure>
+
+![Travel heading along a segment oriented due east](/img/transportation/heading.svg)
+
+<figcaption>
+
+*Travel heading along a segment.*
+
+</figcaption>
+</figure>
+
+</div>
+
+### Directionality
+
+<!-- TODO: Update this section to align with:
+
+          https://github.com/OvertureMaps/schema-wg/issues/134
+-->
+
+
+🚧 We are developing a segment-level directionality concept similar to
+[lane directionality](roads#lane-directionality) to indicate what travel
+headings are allowed or prohibited along the segment. This effort is
+ongoing, so please check back soon.
+
+### Sub-types
+
+Segment features have a `subType` property whose value gives more
+specific information about the segment's role within the transportation
+network.
+
+The `subType` property may be one of `road`, `rail`, or `water` but only
+`road` is currently well defined. A `road` segment models any kind of
+road, street, or trail, including a dedicated path for walking, cycling,
+and like activities. For more information about `road` segments, see
+the page on [Roads](roads).
+
+### Level (Z-order)
+
+Segment geometry is two-dimensional. In the real, 3D, world, however the
+entities represented by segments can be above or below each other, as
+may happen with tunnels, bridges, overpasses, and stacked multi-level
+highway interchanges. To accurately render "top-down" 2D maps, it is
+important to know the relative stacking order, or Z-order, of segments.
+
+Segment Z-order is given by the `level` property. A `level` value of
+`0` indicates visual level, with positive numbers indicating above
```

Comment:
is 'visual level' ~= to 'ground or surface level' or is it more just a somewhat default 'neutral level' that everything is generally above (positive levels) or below (negative levels).

Reply:
I think I get it, and don't have a better suggestion, so its not a blocker from my perspective, just wanted to check my understanding

Reply:
I think the former: ~(ground or surface) level.